realm-annotations-processor/src/main/java/io/realm/processor/RealmProxyClassGenerator.java

```diff
@@ -79,7 +86,7 @@ public RealmProxyClassGenerator(ProcessingEnvironment processingEnvironment, Cla
         NULLABLE_JAVA_TYPES.put("byte[]", "new byte[0]");
     }
 
-    private static final Map<String, String> JAVA_TO_COLUMN_TYPES;
+    protected static final Map<String, String> JAVA_TO_COLUMN_TYPES;
```

Comment:
If we want to keep this as strict as possible it might be package protected instead of protected. There are no subclasses for `RealmProxyGenerator`